### DISCUSSION TITLE
GitHub Pages: can I use a mix of html and md files?

### DISCUSSION THREAD

**Discussion by yanamal** [community/community#22648] [Other Feature Feedback, Questions, & Ideas]
<p>In the past, I’ve used GitHub pages to ether:</p>
<ul>
<li>host simple client-side web pages<br>
OR</li>
<li>host .md files and have jekyll automatically convert them to .html</li>
</ul>
<p>It seems like I don’t have to configure anything differently for those two cases; it just works.</p>
<p>However, now I would like to do <strong>both</strong> in one project - I am hosting an HTML JS CSS “web app” of sorts (from the root folder of the master branch of the project), and I’d also be able to access the README.md file as HTML on the github. io page.</p>
<p>But it seems like Jekyll just ignores the .md files if there’s already an index.html - is that correct? or did I just screw up some settings?  I tried adding some default things to <code>_config.yml</code> but it seems to make no difference.</p>
<p>The repository in question is  [redacted because I can only put two links in one post and you can deduce it from the others]</p>
<p>I can visit <a href="https://yanamal.github.io/DocPrompter/README.md" rel="nofollow noopener">https://yanamal.github.io/DocPrompter/README.md</a> and it shows a plain-text representation of the markdown file, but <a href="https://yanamal.github.io/DocPrompter/README.html" rel="nofollow noopener">https://yanamal.github.io/DocPrompter/README.html</a> is a 404</p>
<p>Thanks!</p>

**Comment by yanamal** [comment, marked as answer]
<p>The problem turned out to be much simpler than I thought. Through experimentation, I found that jekyll seems to just exclude README.md specifically from the pages it turns into HTML.</p>
<p>So I ended up creating a separate .md file and just using the following line to include the contents of README.md in it:</p>
<pre><code class="lang-auto">{% include_relative README.md %}
</code></pre>

**Reply by Adpocalyptic** [reply]
Would be really nice to be able to have README.html files instead of just markdown
